lib/broadway_dashboard/metrics.ex

```diff
@@ -147,13 +176,18 @@ defmodule BroadwayDashboard.Metrics do
   @impl true
   def terminate(_reason, state) do
     if state.timer, do: Process.cancel_timer(state.timer)
+    if state.shutdown_timer, do: Process.cancel_timer(state.shutdown_timer)
```

Comment:
We don't need to cancel timers on terminate (they will be recycled automatically if the process dies).

Reply:
Cool! I didn't know that.